File: Programmers.co.kr/Level 2/주식가격/solution.py
def solution(prices):
    answer = []
    
    for i in range(len(prices) - 1):
        answer.append(0)
        for j in range(i + 1, len(prices)):
            answer[i] += 1
            if prices[i] > prices[j]:
                break

    answer.append(0)
    return answer

File: Programmers.co.kr/Level 2/주식가격/solution.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> prices)
{
    vector<int> answer;
    
    for(int i = 0; i < prices.size() - 1; i++)
    {
        answer.push_back(0);
        
        for(int j = i + 1; j < prices.size(); j++ )
        {
            answer[i]++;
            if(prices[i] > prices[j])
            {
                break;
            }
        }
    }
    
    answer.push_back(0);
    return answer;
}
